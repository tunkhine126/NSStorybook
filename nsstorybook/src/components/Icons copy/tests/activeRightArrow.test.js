/**
 *
 * Tests for ActiveRightArrow
 *
 * @see https://github.com/react-boilerplate/react-boilerplate/tree/master/docs/testing
 *
 */

import React from 'react';
import TestingComponent from 'components/shared/Testing';
import { render } from '@testing-library/react';
// import 'jest-dom/extend-expect'; // add some helpful assertions
import ActiveRightArrow from '../activeRightArrow';

describe('<ActiveRightArrow />', () => {
  it('Expect to not log errors in console', () => {
    const spy = jest.spyOn(global.console, 'error');
    render(
      <TestingComponent>
        <ActiveRightArrow />
      </TestingComponent>
    );
    expect(spy).not.toHaveBeenCalled();
  });

  it('Should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <TestingComponent>
        <ActiveRightArrow />
      </TestingComponent>
    );
    expect(firstChild).toMatchSnapshot();
  });
});
