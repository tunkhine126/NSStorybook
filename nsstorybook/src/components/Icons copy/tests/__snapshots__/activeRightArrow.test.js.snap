// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ActiveRightArrow /> Should render and match the snapshot 1`] = `
<svg
  fill="none"
  height="12"
  viewBox="0 0 7 12"
  width="7"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    clip-rule="evenodd"
    d="M1.52595 11.25C1.57133 11.2469 1.58288 11.2473 1.62781 11.2395C1.72883 11.2221 1.826 11.1838 1.91173 11.1276C1.95508 11.0991 1.99292 11.0661 2.03072 11.0308L6.53092 6.53057C6.56197 6.49731 6.57063 6.48973 6.59819 6.45339C6.62913 6.41259 6.65587 6.3686 6.67788 6.32233C6.77404 6.12016 6.77404 5.88026 6.67788 5.67809C6.65587 5.63181 6.62913 5.58782 6.59819 5.54702C6.57063 5.51072 6.56197 5.5031 6.53092 5.46988L2.03072 0.969684C1.99746 0.938633 1.98985 0.929933 1.95354 0.902407C1.85822 0.830066 1.74571 0.781239 1.62781 0.760875C1.54361 0.7464 1.4571 0.7464 1.37291 0.760875C1.30559 0.772501 1.23993 0.793352 1.17824 0.822678C0.943926 0.934095 0.780794 1.16522 0.754243 1.4233C0.74378 1.52531 0.754468 1.62919 0.785482 1.72692C0.811321 1.80833 0.85111 1.88517 0.902712 1.9532C0.930275 1.98954 0.938939 1.99712 0.96999 2.03038L4.93983 6.00023L0.96999 9.97007L0.935039 10.0075C0.90635 10.0427 0.898174 10.051 0.873198 10.0891C0.826359 10.1605 0.791857 10.2399 0.771644 10.3228C0.75548 10.3892 0.74843 10.4577 0.750755 10.526C0.759606 10.7853 0.906612 11.027 1.13275 11.1542C1.20719 11.196 1.28875 11.225 1.37291 11.2395C1.41783 11.2473 1.42938 11.2469 1.47476 11.25C1.49182 11.25 1.50889 11.25 1.52595 11.25Z"
    fill="#00BAE0"
    fill-rule="evenodd"
  />
</svg>
`;
